### DISCUSSION TITLE
interest for a `oras search`?

### DISCUSSION THREAD

**Discussion by rchincha** [oras-project/oras#1257] [Ideas]
CNCF artifacts WG is looking to standardize search/query mechanism.
https://github.com/cncf-tags/wg-artifacts/pull/34

While this addresses the server-side of the story, support is still needed from client-side.

https://oras.land/docs/category/oras-commands/

**Comment by rchincha** [comment]
Note that the `zot` project already supports these, however they are non-standard and specific to zot.
https://github.com/project-zot/zot/blob/main/pkg/extensions/search/search.md

**Comment by FeynmanZhou** [comment]
Hi @rchincha , thanks for considering ORAS as a client implementor for the standard search API support. 
I will bring this topic to next community meeting and talk with other @oras-project/oras-maintainers on this proposal.

We can keep an eye on the Search API proposal in wg-artifact.

**Reply by rchincha** [reply]
@FeynmanZhou thanks for engaging. We are fleshing out details with community participation (feel free to join pls), but nothing has as much impact as an implementation folks can lay their hands on.
